message.services.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  plugin.manager.message.purge:
+    class: Drupal\message\MessagePurgePluginManager
+    parent: default_plugin_manager
+  message.purger:
```

Comment:
That's a good idea. I thought all the logic would sit inside `MessagePurgePluginManager` including the actual purging, but this separation makes sense to me. 👍